install/tools/ipa-replica-conncheck

```diff
@@ -346,9 +346,9 @@ class PortResponder(threading.Thread):
                 sock.listen(1)
 
             logger.debug('%d %s: Started listening', port, proto)
-        except socket.error as e:
+        except socket.error:
             logger.warning('%d %s: Failed to bind', port, proto)
-            logger.debug("%s", traceback.format_exc(e))
+            logger.debug("%s", traceback.format_exc())
```

Comment:
Since the entire code uses `%s`, I would just go with it. I suppose that it is actually how standard python logging should be used, although I am not that sure about that.

So I would just go with it.